src/webview/create-component/createComponentLoader.ts

```diff
@@ -268,7 +240,7 @@ function getCompDescription(devfiles: AnalyzeResponse[]): ComponentTypeDescripti
 function deleteFile(file: string): Promise<boolean> {
     return new Promise<boolean>(function (resolve, _reject) {
         try {
-            fs.unlinkSync(file)
+            fs.unlink(file)
```

Comment:
You should be able to replace all occurences of `deleteFile(...` with `await fs.unlink(...`